README.md

```diff
@@ -1,2 +1,68 @@
 # Mathemagician App
 
+## Built With
+
+- HTML
+- css
+- javascript
+- React js
+
+## Live Demo 
+
+Not available.
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+A web browser (e.g. Mozilla Firefox, Google Chrome etc.)
+
+git: This is a version control system used for source code management.
+
+A text editor (e.g Visual Studio Code (recommended), Sublime Text, Atom etc.)
+
+Live Server: A web extension utility for Visual Studio Code that helps you live reload feature for dynamic pages.
+
+Basic knowledge on Html, Css, Javascript and React js.
+
+### Setup
+With git, clone the code to your machine, or download the files directly.
+
+git clone https://github.com/leonard33/mathmagicians.git
+
+Once the files are on your machine, open the Mathemagician folder in Visual Studio Code.
+
+### Deployment
+After opening the files in Visual Studio Code, Go to your terminal and writ comand npm start to launch the files with your browser.
```

Comment:
- [ ] (OPTIONAL) Kindly put the `npm install` command before the `npm start` since you can't start the server without installing the dependencies 👍